main/zoe/api/zoe-contract-facet.md

```diff
@@ -259,13 +259,23 @@ to manipulate the offer. The queries and operations are as follows:
         Price: moolaAmountMath.make(9)
       }
       ```
- - `exit()`
+ - `exit(completion)`
    - Returns: `void`
    - Causes the `seat` to exit, concluding its existence. All `payouts`, if any,
-     are made, and the `seat` object can no longer interact with the contract. .
- - `kickOut(msg)` (`msg` is optional)
+     are made, and the `seat` object can no longer interact with the contract.
+     The `completion` argument is usually a string, but this is not required. Its
+     only use is for the notification sent to the contract instance's `done()` function. 
+     Any still open seats or other outstanding promises are closed with a generic 'vat terminated' 
+     message.
```

Comment:
Sorry, I wasn't clear. 

This applies to `shutdown()`, but not to `exit()` or `fail()`. When you exit/fail an individual seat, that seat is done, but the contract continues, and any other seats continue. When you shutdown the vat & contract instance, any remaining seats have `fail()` called. Normally, one wouldn't call shutdown unless all the seats had already completed their work.